go.mod

```diff
@@ -67,6 +67,7 @@ exclude (
 	k8s.io/legacy-cloud-providers v0.0.0
 	k8s.io/metrics v0.0.0
 	k8s.io/mount-utils v0.0.0
+	k8s.io/pod-security-admission v0.0.0
```

Comment:
Just wondering where this comes from since I don't see it being directly used and isn't part of the indirect dep list.

Reply:
It was introduced in v1.22 as a dependency for K8s. https://pkg.go.dev/k8s.io/pod-security-admission/admission?tab=versions

So, with out having this exclude in there it would throw an error run `go mod tidy`, similar to what occurs if we don't exclude the other libraries.

Reply:
Ah that makes. If I recall correctly Pod Security Policies were removed in 1.22 and replaced with Pod Security Admissions